<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanelWindowWidget</class>
 <widget class="QDialog" name="AdminPanelWindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Панель админа</string>
  </property>
  <widget class="QTableWidget" name="clientsTableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>611</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="contractsTableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>611</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exitPushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
